config.py

```diff
@@ -0,0 +1,28 @@
+from enum import Enum
+from logging import ERROR
+
+DOTENV_PATH = './.env'
+OUTPUT_PATH = 'results/'
+MODELS_PATH = 'models/'
+
+
+class API_Status(int,Enum):
```

Comment:
Theses constants are already implemented in Flask

Reply:
The constants are for internal configuration. EG a bid offer request can be made after COB with valid payload and header but since offers has closed for the day, the transaction is pegged for the following day. The HTTP response code is 201, API status is 099 and message is "Transaction for Next Business Day". 

So every response (200-400) returns the below payload structure

```
"status_code": 100,
  "status_message": "Executed Successfully",
  "data": "[12.32062341 18.99951441]"
```